mods/lord/lord_bows/init.lua

```diff
@@ -0,0 +1,67 @@
+-- lord bows
+GRAVITY = 9.81
```

Comment:
оказывается есть вот такая настройка:
```suggestion
GRAVITY = minetest.settings:get("movement_gravity")) or 9.81
```